### ISSUE TITLE
Source maps just started failing to upload

### ISSUE THREAD

**Issue by dwilt** [bugsnag/bugsnag-sourcemaps#42]
### Expected behavior
Source maps upload without issues

### Observed behavior
We currently use AppCenter (and have been for a while successfully) to upload our sourcemaps to bugsnag after a build. Our `appcenter-post-build.sh` file looks like this: 

```sh
bugsnag-sourcemaps upload \
--api-key [KEY GOES HERE - HIDDEN FOR PRIVACY] \
--app-version $APPCENTER_BUILD_ID \
--minified-file ios-release.bundle \
--source-map ios-release.bundle.map \
--minified-url main.jsbundle \
--upload-sources
```

Just today, we've been getting this error when uploading:

```sh
/Users/vsts/.nvm/versions/node/v8.16.0/bin/react-native -> /Users/vsts/.nvm/versions/node/v8.16.0/lib/node_modules/react-native-cli/index.js
+ react-native-cli@2.0.1
+ bugsnag-sourcemaps@1.2.0
added 127 packages from 52 contributors in 6.644s
Loading dependency graph, done.
info Writing bundle output to:, ios-release.bundle
info Writing sourcemap output to:, ios-release.bundle.map
info Done writing bundle output
info Done writing sourcemap output
warn Assets destination folder is not set, skipping...
[34m[info][39m Uploading (ios-release.bundle.map)
[31m[error][39m Error uploading source maps: Unknown read stream path of type "string"
##[error]/bin/bash failed with return code: 1
##[error]Bash failed with error: /bin/bash failed with return code: 1
##[section]Finishing: Post Build Script
##[section]Starting: Xcode build (signed)
```

### Steps to reproduce
*[Insert reproduction steps (if known)]*

### Version
We're using the latest since every time we spin up a build, we do a fresh install of `bugsnag-sourcemaps` via `npm i -g react-native-cli bugsnag-sourcemaps`

### Additional information

`appcenter-post-build.sh`
```sh
# Generate sourcemaps for Bugsnag

npm i -g react-native-cli bugsnag-sourcemaps

## Create sourcemaps and bundle
react-native bundle \
--platform ios \
--dev false \
--entry-file index.js \
--bundle-output ios-release.bundle \
--sourcemap-output ios-release.bundle.map

## Upload sourcemaps to Bugsnag
bugsnag-sourcemaps upload \
--api-key [KEY] \
--app-version $APPCENTER_BUILD_ID \
--minified-file ios-release.bundle \
--source-map ios-release.bundle.map \
--minified-url main.jsbundle \
--upload-sources
```

#### Can't comment on Issues?
Some users have been unable to comment on Github issues when an [adblocker extension is enabled](https://docs.bugsnag.com/platforms/browsers/faq/#is-bugsnag-blocked-by-ad-blockers).
We recommend temporarily disabling the extension, or if that fails, contacting support@bugsnag.com.

**Comment by dwilt** [comment]
FYI: I saw that [1.2.0](https://github.com/bugsnag/bugsnag-sourcemaps/commit/9131a997daee4186871f1b523a3515efebe74421) was released shortly ago so I rolled back the version in our script to `1.1.0` and our uploads started working again.

**Comment by mayank-shaan** [comment]
+1 Having the same issue since today

**Comment by bengourley** [comment]
Hey all – apologies for this. Please pin your version to `1.1.0` for now. I've identified the fix and we should be shipping a new version shortly.

**Comment by bengourley** [comment]
While we QA the fix on #43, it's worth noting that `react-native bundle` includes the sources in the source map under the `sourcesContent` property. This means that you don't need to use the `--upload-sources` option. If you do, it unnecessarily includes the source content twice – once in the source map and once as parameters in the upload body.

tl;dr only use `--upload-sources` if you have to!

**Comment by bengourley** [comment]
This is fixed in [v1.2.1](https://github.com/bugsnag/bugsnag-sourcemaps/releases/tag/v1.2.1).
